tests/datasets/test_events_processor.py

```diff
@@ -4,55 +4,52 @@
 from collections import OrderedDict
 from datetime import datetime, timedelta
 
-from base import BaseTest, get_event
+from base import BaseEventsTest
 
-from snuba import processor, settings
-from snuba.processor import InvalidMessageType, InvalidMessageVersion, process_message
+from snuba import settings
+from snuba.datasets.events_processor import InvalidMessageType, InvalidMessageVersion
 
 
-class TestProcessor(BaseTest):
-    def setup_method(self, test_method):
-        super(TestProcessor, self).setup_method(test_method, 'events')
-        self.event = get_event()
-
+class TestEventsProcessor(BaseEventsTest):
     def test_simple(self):
-        _, processed = process_message(self.dataset.get_promoted_tag_columns(), self.event)
```

Comment:
do we still need to expose get_promoted_tag_columns on the dataset ? Or can we jsut remove the method or making it private ?

Reply:
It's used in one last place right now, will remove it completely later in https://github.com/getsentry/snuba/pull/346